lib/email_checker/loader.ex

```diff
@@ -1,16 +1,37 @@
 defmodule EmailChecker.Loader do
   use Application
 
-  def default_dns do
-    Application.get_env(:email_checker, :default_dns) || {4, 2, 2, 1}
+  defp default_dns do
+    Application.get_env(:email_checker, :default_dns, :system)
+  end
+
+  defp also_dns do
+    Application.get_env(:email_checker, :also_dns, [])
+  end
+
+  defp insert_ns(ns = {w,x,y,z})
+    when is_integer(w) and is_integer(x) and is_integer(y) and is_integer(z)
+  do
+    # HACK: add a default and "also" DNS when `inet_db` is not yet available
```

Comment:
Do you think this comment should be in this function? 
Default use this function, that's true, but it could be any DNS. We also talk about also_dns in the comment but also_dns use append and not insert.

What do you think about moving this comment back to the start function (where we actually use these 2 values)? Or we could even get rid of the comment? With the new implementation it doesn't look too much like a hack :)